pages/index.vue

```diff
@@ -42,62 +43,86 @@
                     >
                         {{ $t('callForProposals') }}
                     </text-button>
-                </div> -->
+                </div>
             </div>
         </div>
 
-        <!--<i18n-page-wrapper class="sponsor-section pt-12">
-            <h1 class="sponsor-title">{{ $t('sponsorList') }}</h1>
-            <sponsor-card-collection
-                v-for="(leveledSponsors, i) in sponsorsData"
-                :key="`index_sponsor_level_${i}`"
-                :level-name="leveledSponsors.level_name"
-            >
-                <sponsor-card
-                    v-for="(sponsor, j) in leveledSponsors.sponsors"
-                    :key="`index_sponsor_level_${i}_sponsor_${j}`"
-                    :sponsor-name="sponsor.name_en_us"
-                    :logo-url="sponsor.logo_url"
-                    :tag="getAttributeByLocale(sponsor, 'subtitle')"
-                    @click.native="showModal(sponsor)"
+        <i18n-page-wrapper>
```

Comment:
I just discovered that the i18nPageWrapper is not imported and registered on this page, so it actually does not take any effects here. @josix Should the wrapper be removed?

Reply:
Oh, let me fix it. I'll register the `i18nPageWrapper ` and also tune the margin and padding of the content.